package.json

```diff
@@ -55,12 +55,12 @@
     "nmr-correlation": "2.3.3"
   },
   "dependencies": {
-    "@atlaskit/pragmatic-drag-and-drop": "^1.7.3",
-    "@blueprintjs/core": "^5.19.0",
-    "@blueprintjs/icons": "^5.22.0",
-    "@blueprintjs/select": "^5.3.20",
+    "@atlaskit/pragmatic-drag-and-drop": "^1.7.4",
+    "@blueprintjs/core": "^6.0.0",
+    "@blueprintjs/icons": "^6.0.0",
+    "@blueprintjs/select": "^6.0.0",
```

Comment:
Are you sure it's safe? We haven't updated react-science yet: https://github.com/zakodium-oss/react-science/issues/924

Reply:
wait for a new version of react-science

Reply:
I just merged the release